fixtures/generate.js

```diff
@@ -0,0 +1,98 @@
+import fs from 'fs';
+import path from 'path';
+import config from '../yeep.config';
+import { generateFakeUser } from './models/users';
+import { generateFakeOrg } from './models/orgs';
+import { generateFakePermissions } from './models/permissions';
+import { generateFakeRoles } from './models/roles';
+import { promisify } from 'util';
+
+const writeFileAsync = promisify(fs.writeFile);
+
+const generateManyFakeUsers = (numberOfUsers) => {
+  const users = [];
+  // usernames are taken from the email property so both are unique
+  const uniqueUsernames = new Set();
+  for (var i = numberOfUsers - 1; i >= 0; i--) {
+    // maintain some sane loops
+    let count = 0;
+    while (count < 100) {
+      let user = generateFakeUser();
+      if (!uniqueUsernames.has(user.username)) {
+        uniqueUsernames.add(user.username);
+        users.push(user);
+        break;
+      }
+      count ++;
+    }
+  }
+
+  uniqueUsernames.clear();
+  return users;
+}
+
+const generateFakeOrgs = (numberOfOrgs) => {
```

Comment:
Since we changed the name above, can we also change `generateFakeOrgs` to `generateManyFakeOrgs`?